compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -469,7 +469,9 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
                         let count = generics
                             .params
                             .iter()
-                            .filter(|param| matches!(param.kind, ast::GenericParamKind::Lifetime { .. }))
```

Comment:
Could you undo this change?

Reply:
Ok.
Apparently this is how nightly `rustfmt` works (from my editor), but `x.py fmt` doesn't change it back for some reason.

Reply:
Probably we need to update the fmt used by x.py fmt, in stage0.txt. Filed https://github.com/rust-lang/rust/pull/80843 to do that.